src/transaction/lock_manager.h

```diff
@@ -154,7 +154,7 @@ typedef enum
   LOCK_RESOURCE_INSTANCE,	/* An instance resource */
   LOCK_RESOURCE_CLASS,		/* A class resource */
   LOCK_RESOURCE_ROOT_CLASS,	/* A root class resource */
-  LOCK_RESOURCE_OBJECT		/* An object resource */
+  LOCK_RESOURCE_OBJECT		/* An object resource. Obsolete */
```

Comment:
How about change the following too?

case LOCK_RESOURCE_OBJECT: -> assert(); in the lock_res_key_copy()
```
839     case LOCK_RESOURCE_OBJECT:
840       /* nothing, it's a free object */
841       break;
842
843     default:
844       assert (false);
845       return ER_FAILED;
```

Reply:
When the function LOCK_res_key_copy() is called, if the type is LOCK_RESOURCE_OBJECT, then something is wrong. Because this function is called to insert RESOURCE BLOCK.
so It looks good to use fall through for assert.

```
     case LOCK_RESOURCE_OBJECT:
     default:
       assert (false);
       return ER_FAILED;
```

Reply:
@Yechan0815 Do it please.